server/master/src/main/java/org/apache/accumulo/master/metrics/Metrics2ReplicationMetrics.java

```diff
@@ -58,6 +59,7 @@ protected void snapshot() {
     registry.add(PENDING_FILES, getNumFilesPendingReplication());
     registry.add(NUM_PEERS, getNumConfiguredPeers());
     registry.add(MAX_REPLICATION_THREADS, getMaxReplicationThreads());
+    registry.add(LATENCY, getLatencyInSeconds());
```

Comment:
I agree there are better ways of getting a better metric. This approach is a 90%(maybe to generous) solution using the information that already exists to give an end user some bearing of confidence as to current replication latency. From a users perspective, they want a historical way to know what is the greatest latency they can expect their data to be replicated at a given point in time. Spikes help explain the possible -1% unhealthy replicated files. As in the picture above there are couple spikes, but they don't seem to paint a bad picture. I will look into the histogram class. Are there any points of reference in the code you can point me to?